angular-ngrx-scss/src/app/home/profile/profile-repos/profile-repos.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Would potentially be good to update this with a small test on the time functionality you did? Otherwise, can be deleted.